WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -756,13 +756,18 @@ public void onGoogleSignupFinished(String name, String email, String photoUrl, S
 
     @Override
     public void onGoogleSignupError(String msg) {
-        BasicFragmentDialog dialog = new BasicFragmentDialog();
-        dialog.initialize(GOOGLE_ERROR_DIALOG_TAG, getString(R.string.error),
-                msg,
-                getString(org.wordpress.android.login.R.string.login_error_button),
-                null,
-                null);
-        dialog.show(this.getSupportFragmentManager(), GOOGLE_ERROR_DIALOG_TAG);
+        // allow state loss
```

Comment:
Not too important but, I'm kinda puzzled by the comment @malinajirka . If I understand it correctly, the logic here is to avoid showing the dialog because that will cause a FragmentManager state change and we should avoid that ([as per the docs](https://developer.android.com/reference/android/support/v4/app/FragmentManager.html#isStateSaved())). So, I'm not sure which state are we allowing to get lost. It doesn't seem that we are going to commit a transaction with allowing state loss. Am I missing something?

Reply:
> or implement another fix

I'm cool with the actual solution here so, no need for more effort 👍.

Apparently, It's the comment that threw me off here since it's not a "commit with state loss" operation that happens below. So, yeah, let's just rephrase the comment. Maybe something along the lines "Only show the error dialog if the activity is still active", or something.

Reply:
👍